internal/handshake/xor_nonce_aead_boring.go

```diff
@@ -38,10 +37,10 @@ func allZeros(nonce []byte) bool {
 }
 
 func (f *xorNonceAEAD) sealZeroNonce() {
-	f.doSeal([]byte{}, zeroNonce[:], []byte{}, []byte{})
+	f.doSeal([]byte{}, []byte{}, []byte{}, []byte{})
 }
 
-func (f *xorNonceAEAD) seal(out, nonce, plaintext, additionalData []byte) []byte {
+func (f *xorNonceAEAD) seal(nonce, out, plaintext, additionalData []byte) []byte {
```

Comment:
for reference: 
- https://github.com/chungthuang/quic-go/blob/cloudflare/internal/handshake/xor_nonce_aead_noboring.go
- https://github.com/chungthuang/quic-go/blob/cloudflare/internal/handshake/cipher_suite.go#L90